python/rmm/rmm/_lib/device_buffer.pyx

```diff
@@ -138,6 +138,31 @@ cdef class DeviceBuffer:
         }
         return intf
 
+    def prefetch(self, device=None, stream=None):
+        """Prefetch buffer data to the specified device on the specified stream.
+
+        Assumes the storage for this DeviceBuffer is CUDA managed memory
+        (unified memory). If it is not, this function is a no-op.
+
+        Parameters
+        ----------
+        device : optional
+            The CUDA device to which to prefetch the memory for this buffer.
+            Defaults to the current CUDA device. To prefetch to the CPU, pass
+            `~cuda.cudart.cudaCpuDeviceId` as the device.
```

Comment:
It seems the doc build did not render it right. Possibly because the default role in RMM is "cpp" instead of "py". If so the fix would be
```
:py:`~cuda.cudart.cudaCpuDeviceId`
```

Reply:
I am certain that the correct reference is there. It can be checked as follows:
```shell
$ python -m sphinx.ext.intersphinx https://nvidia.github.io/cuda-python/objects.inv | grep "cuda.cudart.cudaCpuDeviceId"
    cuda.cudart.cudaCpuDeviceId                                                      : module/cudart.html#cuda.cudart.cudaCpuDeviceId
```

Reply:
Did this get fixed or should this be raised in a new issue?

Reply:
I don’t think this was seen after merging. Issue filed: https://github.com/rapidsai/rmm/issues/1635#issue-2442423568

Reply:
Fix: https://github.com/rapidsai/rmm/pull/1636#issue-2442432052

Reply:
Thanks Bradley! 🙏